CHANGES.rst

```diff
@@ -5,7 +5,9 @@ CHANGES
 2.2.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Fix list of supported Python versions in Trove classifiers.
```

Comment:
This should mention specific versions

- Added support for Python 3.7
- Dropped support for Python 3.4 and 3.5

Reply:
Oh, actually the older changelog entry for 2.2.0 mentioned dropping Python 3.5 (but forgot to mention 3.4)!   And adding 3.7 (but it wasn't actually added to travis)!

What a mess.